### DISCUSSION TITLE
Will LAMMPS be updated to a newer version?

### DISCUSSION THREAD

**Discussion by iquasere** [nufeb/NUFEB-2#10] [Ideas]
Hey there!

Now that the tool is being revised, will the LAMMPS it depends on be updated to the most recent stable version currently available? Or is the 2020 version enough for the needs of NUFEB?

I understand a lot of work would have to be invested into reformulating NUFEB's code, might not be worth the hassle.

**Comment by shelllbw** [comment]
Hey, I'm planning to update the LAMMPS version before releasing NUFEB-dev code (NUFEB-2) in a couple of months. It's a pain to do such update and I don't see any urgent need at the moment. Let's leave it a bit later : )

ps: upcoming feature will be the energy growth model
